### PR TITLE
Fix Pinot response util to exclude closed wf fields when list open

### PR DESCRIPTION
**Pull request by neil-xie**
**What changed?**
Pinot has default value -1 for CloseTime/CloseStatus when workflow is open

**Why?**
Bug fix

**How did you test it?**
Manual test

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: common/pinot/responseUtility.go [hunk 1]
```diff
@@ -107,10 +107,10 @@ func ConvertSearchResultToVisibilityRecord(hit []interface{}, columnNames []stri
 		ShardID:          source.ShardID,
 		SearchAttributes: attributeMap,
 	}
-	if source.UpdateTime != 0 {
+	if source.UpdateTime > 0 {
```
Comment (taylanisikdemir) [inline common/pinot/responseUtility.go:110]: please add unit tests and also rename file to snake_case

Comment (bowenxia) [reply common/pinot/responseUtility.go:110]: This file was factored out from pinot_client.go. And I've added unit tests for this file in pinot_client_test.go

Comment (taylanisikdemir) [reply common/pinot/responseUtility.go:110]: nice! we can add test case for -1

Comment (neil-xie) [reply common/pinot/responseUtility.go:110]: Sure, I added a new test file for response_utility and moved the test method there. Also added new test case

**Review by bowenxia — APPROVED** [review]
LGTM

File: common/pinot/responseUtility.go (hunk 1 above)
Comment (taylanisikdemir) [reply common/pinot/responseUtility.go:110]: thanks Neil.

**Pull request merged**